### DISCUSSION TITLE
Proximity of observation data

### DISCUSSION THREAD

**Discussion by scadergit** [open-meteo/open-meteo#161] [General]
For locations with sparse stations (e.g. US West), does the API do any interpolation to determine the current temperature, or is it simply the nearest station even if that is distance is greater than average?

**Comment by patrick-zippenfenig** [comment]
Right now, all current weather information is based on weather models

For North America the GFS HRRR model is used, which updates every hour with a resolution of 3 kilometre. Weather models are based on measurements. The first hour of each weather model is an analysis of all available observations. Thus the current weather information is some kind of interpolated measurements.

Typically this results in a far more consistent current weather information, because a lot of false measurement are removed already

As all data is gridded to 3 km, there is no fixed proximity to measurement station. Ideally there are multiple weather stations in each grid cell, but there could also be none.
